apps/extension/src/routes/page/onboarding/set-password.tsx

```diff
@@ -20,6 +20,15 @@ export const SetPassword = () => {
   const [password, setPassword] = useState('');
   const [confirmation, setConfirmation] = useState('');
 
+  const handleSubmit = (event: FormEvent | MouseEvent) => {
+    event.preventDefault();
+
+    void (async () => {
+      await finalOnboardingSave(password);
+      navigate(PagePath.ONBOARDING_SUCCESS);
+    })();
+  };
```

Comment:
Yeah — a form logically groups everything together so that, e.g., pressing <kbd>Enter</kbd> inside one of the text fields runs the form's `onSubmit` function.